source/catalog.html

```diff
@@ -40,83 +39,85 @@
+            <h3 class="description-photo__title">Неродные просторы</h3>
+            <p class="description-photo__text">Автор фото: Борис</p>
+            <span class="description-photo__like">1350</span>
           </div>
         </li>
-        <li class="foto__item">
-          <img class="foto__image" src="#" width="0" height="0" alt="Название фото">
-          <div class="foto__wrapper discription-foto">
-            <h3 class="discription-foto__title">Местная растительность</h3>
-            <p class="discription-foto__text">Автор фото: Сергей</p>
-            <span class="discription-foto__span discription-foto__span--like">143</span>
+        <li class="photo__item">
+          <img class="photo__image" src="#" width="0" height="0" alt="Название фото">
+          <div class="photo__wrapper description-photo">
+            <h3 class="description-photo__title">Местная растительность</h3>
+            <p class="description-photo__text">Автор фото: Сергей</p>
+            <span class="description-photo__like">143</span>
           </div>
         </li>
-        <li class="foto__item">
-          <img class="foto__image" src="#" width="0" height="0" alt="Название фото">
-          <div class="foto__wrapper discription-foto">
-            <h3 class="discription-foto__title">Дорога на север</h3>
-            <p class="discription-foto__text">Автор фото: Петр</p>
-            <span class="discription-foto__span discription-foto__span--like">96</span>
+        <li class="photo__item">
+          <img class="photo__image" src="#" width="0" height="0" alt="Название фото">
+          <div class="photo__wrapper description-photo">
+            <h3 class="description-photo__title">Дорога на север</h3>
+            <p class="description-photo__text">Автор фото: Петр</p>
+            <span class="description-photo__like">96</span>
           </div>
         </li>
-        <li class="foto__item">
-          <img class="foto__image" src="#" width="0" height="0" alt="Название фото">
-          <div class="foto__wrapper discription-foto">
-            <h3 class="discription-foto__title">Мост дьявола</h3>
-            <p class="discription-foto__text">Автор фото: Антон</p>
-            <span class="discription-foto__span discription-foto__span--like">254</span>
+        <li class="photo__item">
+          <img class="photo__image" src="#" width="0" height="0" alt="Название фото">
+          <div class="photo__wrapper description-photo">
+            <h3 class="description-photo__title">Мост дьявола</h3>
+            <p class="description-photo__text">Автор фото: Антон</p>
+            <span class="description-photo__like">254</span>
           </div>
         </li>
       </ul>
     </section>
     <section class="video">
+      <h2 class="visually-hidden">Видеопрезентация маршрутов</h2>
       <header class="video__header">
-        <h2 class="video__title">Все ещё сомневаетесь?</h2>
+        <h3 class="video__title">Все ещё сомневаетесь?</h3>
```

Comment:
Так а зачем он нам в роли заголовка? Это ж просто текст по смыслу, он не озаглавливает, он же не дает понимание о содержании
```suggestion
        <p class="video__title">Все ещё сомневаетесь?</p>
```